internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -195,16 +195,9 @@ func (r *ClusterDeploymentsReconciler) Reconcile(origCtx context.Context, req ct
 		return r.updateStatus(ctx, log, clusterInstall, cluster, err)
 	}
 
-	if clusterDeployment.Spec.PullSecretRef == nil {
-		errorMessage := "Missing reference to pull secret"
-		log.WithError(err).Error(errorMessage)
-		return r.updateStatus(ctx, log, clusterInstall, cluster, newInputError(errorMessage))
-	}
-
-	// Make sure that the PullSecret Secret has the needed label
```

Comment:
Also the comment and error message here didn't match the code - it's not checking or setting a label - fixed it